container-engine-lib/lib/backend_impls/metrics_reporting/metrics_reporting_kurtosis_backend.go

```diff
@@ -2,18 +2,20 @@ package metrics_reporting
 
 import (
 	"context"
+	"io"
```

Comment:
hmm where are `io` and `time` being used/are they needed imports with this change?